curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/6553980e0527fa115c705646.md

```diff
@@ -1,8 +1,8 @@
 ---
 id: 6553980e0527fa115c705646
-title: Step 39
+title: Step 43
 challengeType: 20
-dashedName: step-39
+dashedName: step-43
 ---
 
 # --description--
```

Comment:
Updated instruction:
Now, instead of printing `space!`, use the addition assignment operator to add the space(currently stored in `char`) to the current value of `encrypted_text`.